test/kokoro/xds_k8s.sh

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bash
+# Copyright 2021 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+# Constants
+readonly GITHUB_REPOSITORY_NAME="grpc-go"
+# GKE Cluster
+readonly GKE_CLUSTER_NAME="interop-test-psm-sec-v2-us-central1-a"
```

Comment:
Stuff like this really sets us up for future maintenance headaches, IMO.

How do we know this is the right value to use?

How do we keep this in sync with wherever the other thing is that relies upon this?

Reply:
Yes, this will go away. That's one of primary reasons why we need shared build scripts (b/186553728). At the moment switching the cluster would involve modifying this line in three repos, and in all branches with xds-k8s enabled.